### ISSUE TITLE
Make a civilian structure buildable

### ISSUE THREAD

**Issue by helloisitmeyourelookingfor** [OpenRA/OpenRA#12932]
So im pretty much new to OpenRa. I'm trying to make the Forward Command Post to be buildable for soviets in campaign and in single skrimish, tried changing the civilian.yaml on the rules folder, copying some lines from other soviet structure ,airfield i think, as i wanted it to need a radar first; goes without sating this failed miserably and made the game crash as soon as the MCV is deployed. 

So basically what is the proper procedure to incorporate a structure into a faction and make it buildable?

**Comment by abc013** [comment]
Have you already looked into [that](https://github.com/OpenRA/OpenRA/wiki/Creating-a-new-unit)?

**Comment by MustaphaTR** [comment]
Issue must be that it doesn't have a cameo. You should find a cameo for it from somewhere and define it on */sequences/structures.yaml or sequences part of your map which one you are tring to do.

**Comment by helloisitmeyourelookingfor** [comment]
@MustaphaTR Yep, seems like it was exactly that. All good now, thanks!

Small detail tho , I changed the name on both the /rules/structures and the /rules/civilian .YAMLs to "Forward Command Post" but it still displays "Forward Command" on the game, is it because of a text length limit or do i need to change it on some other file?

**Comment by helloisitmeyourelookingfor** [comment]
@MustaphaTR That was it, worked perfectly. 

Thanks!
